example/src/Main.purs

```diff
@@ -77,15 +77,15 @@ main = onLoad $ do
   Anchor.onChange endAnchor rerenderMarker
 
   Editor.getKeyBinding editor
-    >>= KeyBinding.addKeyboardHandler \{editor} hs kstring kcode evt -> do
+    >>= KeyBinding.addKeyboardHandler \{editor : editor_shadow} hs kstring kcode evt -> do
       if hs == -1 || (kcode <= 40 && kcode >= 37)
         then pure Nothing
         else do
         Ace.Position {row: startRow, column: startColumn}
           <- Anchor.getPosition startAnchor
         Ace.Position {row: endRow, column: endColumn}
           <- Anchor.getPosition endAnchor
-        selectedRange <- Editor.getSelectionRange editor
+        selectedRange <- Editor.getSelectionRange editor_shadow
```

Comment:
Just to quickly check — what’s the reason for this change? Was it not working before?

Reply:
To avoid a shadowed variable error / warning. Another naming option is `editor'`

Reply:
Oh, interesting, I didn’t realize this was shadowed before. Let’s do with editor’ and I can merge.